api_v3/lib/KalturaEntryService.php

```diff
@@ -700,6 +700,13 @@ protected function attachOperationResource(kOperationResource $resource, entry $
 				return $this->attachLiveEntryResource($internalResource, $dbEntry, $dbAsset, $operationAttributes);
 			}
 
+			$srcEntry = self::getEntryFromContentResource($internalResource);
```

Comment:
should be checked when we check for multi clip, if someone tries to multi-clip on a recorded entry we should notify them this operation is not permitted.